src/Shared/AWS/SemanticConventionVersion.cs

```diff
@@ -0,0 +1,90 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+// because this class is shared and public, each project that exposes it must
+// ensure that it has a unique namespace, otherwise users could face type collisions
+// if they import multiple projects.
+#if Instrumentation_AWSLambda
+namespace OpenTelemetry.Instrumentation.AWSLambda;
+#elif Instrumentation_AWS
+namespace OpenTelemetry.Instrumentation.AWS;
+#elif Resources_AWS
```

Comment:
Typically in C# alle markers for conditional compilation are `UPPERCASE`. Consider this for consistency.

Reply:
agreed - changed case.